<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblVideo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>341</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Video Frame</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QCheckBox" name="checkBox_fullFace">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Detect Face</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_eyes">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Detect Eyes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_mouth">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Detect Mouth</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_performPreProcessing">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Perform Pre - Processing</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblProcessFaceFrame">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Face Frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblProcessEyeFrame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Eye Frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblProcessMouthFrame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>450</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>MouthFrame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSnapshot">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Snapshot</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
